semperfiwebdesign/all-in-one-seo-pack
# Change order of Open Graph tags

Issue #2804

## Proposed changes
This changes the order of some of the Open Graph meta tags in accordance with guidelines from Pinterest.

## Types of changes
- Improves existing code

## Checklist

## Testing instructions 
These tests should be performed on a publicly accessible site.
- Enable the Social Meta module and configure all settings in the Facebook Settings and Twitter Settings sections
- Edit your user profile and add a Twitter user and Facebook author in the All in One SEO Pack section
- Create a post and complete all of the settings on the Social Settings tab
- View the source code of the post and check that all of the Open Graph tags are present and in the new order
- Test the post with the Facebook Debug Tool, The Twitter Card Validator and the Pinterest Rich Pins Validator (see links below)

## Further comments
Facebook Debug Tool - https://developers.facebook.com/tools/debug/
Twitter Card Validator - https://cards-dev.twitter.com/validator
Pinterest Rich Pins Validator - https://developers.pinterest.com/tools/url-debugger/
